src/main/java/hudson/plugins/emailext/plugins/recipients/DevelopersRecipientProvider.java

```diff
@@ -29,8 +30,12 @@ public DevelopersRecipientProvider() {
     public void addRecipients(ExtendedEmailPublisherContext context, EnvVars env, Set<InternetAddress> to, Set<InternetAddress> cc, Set<InternetAddress> bcc) {
         ExtendedEmailPublisherDescriptor descriptor = Jenkins.getInstance().getDescriptorByType(ExtendedEmailPublisherDescriptor.class);
         Set<User> users = new HashSet<User>();
-        for (ChangeLogSet.Entry change : context.getBuild().getChangeSet()) {
-            users.add(change.getAuthor());
+
+        if (context.getRun() instanceof AbstractBuild) {
+            AbstractBuild<?, ?> build = (AbstractBuild) context.getRun();
+            for (ChangeLogSet.Entry change : build.getChangeSet()) {
```

Comment:
You can probably use `getChangeSets` via reflection, pending [JENKINS-24141](https://issues.jenkins-ci.org/browse/JENKINS-24141).